android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -475,72 +476,109 @@ public Builder(String appKey, String appSecret, Context context, HttpTransport t
          * </p>
          *
          * @param context - Your Android Application Context
-         * @param transport - Your HttpTransport
          *
          */
-        public Builder(Context context, HttpTransport transport) {
+        public Builder(HttpTransport transport, Context context) {
```

Comment:
@misha-vakulich Is this change to this `Builder` constructor a breaking change from the last beta release?  If so, please make a note of it to mention this in the change log.